rfc/x/index.md

```diff
@@ -0,0 +1,491 @@
+approve the work, building confidence that the community is moving in the
+right direction. More information on this strategy can be found under:
+<https://engineering.squarespace.com/blog/2019/the-power-of-yes-if> .
+
+**Editors** identify whether a PR should or should not follow the RFC process,
+and choosing when a draft is ready to become an RFC. They also choose
+appropriate **Reviewers** for an RFC and manage the communication between
+**Authors** and **Reviewers**. 
+
+Note: The use of "Editors" in this document is intended as a placeholder. A
+future RFC will define the selection and removal of editors. Until that time,
+the sole editor is Josh Moore which has been the _de facto_ case since the
+inception of NGFF. 
+
+**Reviewers** represent experts in the community whose opinion on an RFC is
+necessary in order for the community to feel comfortable that a solution is
+advantageous for and implementable by the community. Some number of reviewers
+will be asked to provide a review of the RFC along with a recommendation on how
+the **Editors** should proceed with the RFC. Depending on the complexity of the
+RFC, this process may be iterative and require multiple reviews.
+
+**Commenters** are other members of the community who are not required to make
+a determination on an RFC but are still invited to add their voice to the
+decision-making process.
+
+## Implementation
+
+This description of the RFC process will refer frequently to the [visual diagram][dia]
+Readers may want to familiar themselves with it at this point.
+Identifiers such as "D1", "R2", "S3", refer to steps in that diagram.
+
+### Phases
+
+The overall process is broken down into three phases: DRAFT phase before a
+proposal becomes an RFC, the RFC phase during which reviews occur, and the SPEC
+phase after the RFC has been adopted.
+
+#### DRAFT
+
+The DRAFT phase begins when **Authors** propose (D1) a new idea and
+subsequently gather support (or "socialize") the idea (D2) before opening a PR
+(D3). This phase is meant to promote ideation without premature criticism, though
+clarifications can be requested on the GitHub PR (D4) which may require an
+update of the PR (D3).
+
+One potential clarification is whether or not a given PR should be an RFC. The
+RFC process is intended primarily for significant and/or breaking changes to
+the specification or the community process itself. Other changes, bug fixes,
+etc. are welcome as PRs without a full RFC. If **Authors** are unsure whether
+or not a change requires an RFC, they can open an issue or contact the **Editors**
+directly. All RFC-targeted PRs SHOULD follow the [current template][tmpl].
+
+> 🕑 The iterative clarification phase should be limited to at most a month.
+
+At the **Editors** discretion (D5), the PR can be merged at which point it
+becomes an RFC or closed if there is no interest in progressing with the
+proposal. In the latter case, **Authors** can use any feedback to open a new PR
```

Comment:
The diagram.png doesn't mention that the `DRAFT PR` gets merged or closed at `D5`. Only `approve Yes/No`. The `PR persists` state doesn't indicate that the PR is closed.
Is there a point at which the `DRAFT PR` loses the "DRAFT" status and becomes a "ready to review" PR, or maybe `DRAFT PR` is a PR to DRAFT and RFC, rather than a PR with "DRAFT" status on github?

It wasn't clear to me from the diagram whether the `RFC` section was reviewing an open PR, but from reading the text here is appears not.

Who is responsible for `D4`? It's the only block that isn't assigned to a stakeholder role. I assume it is `REVIEWERS` but are they invited or notified about the PR by EDITORS?